app/helpers/lessons_helper.rb

```diff
@@ -3,9 +3,9 @@ def github_edit_url(lesson)
     github_link("curriculum/edit/main#{lesson.github_path}")
   end
 
-  def github_report_url(lesson)
+  def github_report_url(lesson, path)
     # rubocop:disable Layout/LineLength
-    github_link("curriculum/issues/new?assignees=nil&labels=Status%3A+Needs+Triage&projects=&template=suggestion.yaml&title=#{lesson.title}%3A+%3CShort+description+of+your+suggestion%3E")
+    github_link("curriculum/issues/new?assignees=nil&labels=Status%3A+Needs+Triage&projects=&template=suggestion.yaml&title=#{lesson.title}%3A+%3CShort+description+of+your+suggestion%3E&lesson-link=https://www.theodinproject.com#{path}")
```

Comment:
Nit: It might read a little better using the lesson_url route helper here 

 ```suggestion
    github_link("curriculum/issues/new?assignees=nil&labels=Status%3A+Needs+Triage&projects=&template=suggestion.yaml&title=#{lesson.title}%3A+%3CShort+description+of+your+suggestion%3E&lesson-link=#{lesson_url(lesson)}")
```

But I don't think theres a wrong answer with whatever method we use to get the url. The lesson_url helper would avoid another argument being needed, but thats not the end of the world. Happy with whatever you like best.

Reply:
Would `lesson_url` also work for non-lesson pages such as about, FAQ, etc? Otherwise I think it would be preferable to not use it so this will work on any page (just in case we later want to use it on other pages).

Reply:
Actually hmmm, on second thought: Those would probably require an entirely different system, so this seems best for now